### ISSUE TITLE
Screen brightness changes while typing

### ISSUE THREAD

**Issue by e-senturk** [black-dragon74/ROG-HID#12]
Some times while typing screen brightness changes itself.

**Comment by e-senturk** [comment]
Also on some boots keyboard not working. Its probably our model is not same with yours. 
KB Model: 0x1866 0x0b05

**Comment by jiahao1889** [comment]
@black-dragon74 yeah on my rog m15 'm' key as brightness down, '3' key as brightness up, strange

**Comment by black-dragon74** [comment]
@e-senturk @jiahao1889 Please attach your IOREG?

**Comment by htmambo** [comment]
@black-dragon74 I also encountered the same problem,this is my ioreg output:
[ioreg.txt](https://github.com/black-dragon74/ROG-HID/files/6952726/ioreg.txt)
